outputs.tf

```diff
@@ -1,11 +1,11 @@
 output "instances_role_arn" {
   description = "ARN of the IAM role of the EC2 instances"
-  value       = aws_iam_role.this.arn
+  value       = aws_iam_role.this.*.arn
```

Comment:
Maybe update the description to mention that this won't be populated if a custom role is specified? I know it seems obvious, but might as well be explicit.

Reply:
Sure.

Reply:
Done